libmscore/articulation.cpp

```diff
@@ -270,6 +272,16 @@ void Articulation::read(XmlReader& e)
             else if (tag == "direction") {
                   setProperty(P_ID::DIRECTION, Ms::getProperty(P_ID::DIRECTION, e));
                   }
+            else if ( tag == "ornamentStyle") {
+                auto text = e.readElementText();
+                if ( text == "Baroque" || text == "1")
+                    setOrnamentStyle(MScore::OrnamentStyle::BAROQUE);
+                else
+                    setOrnamentStyle(MScore::OrnamentStyle::DEFAULT);
+            }
+            else if ( tag == "playArticulation")
```

Comment:
Why having two properties ornamentStyle and playArticulation instead of just ornamentStyle with a value of "don't play"?